src/xz/private.h

```diff
@@ -34,16 +36,28 @@
 #endif
 
 #ifndef STDIN_FILENO
+#ifdef _MSC_VER
+#	define STDIN_FILENO (_fileno(stdin))
```

Comment:
I suggest using `#define fileno _fileno` like in xzdec.c to avoid repetition.

Reply:
Fixed in a new commit.